gerrit/install.sls

```diff
@@ -24,12 +24,36 @@ group_{{ settings.group }}:
     - group: {{ settings.group }}
     - makedirs: true
 
+{{ settings.base_directory }}/{{ settings.site_directory }}/lib:
+  file.directory:
+    - user: {{ settings.user }}
+    - group: {{ settings.group }}
+    - makedirs: true
+
+{% for name, library in salt['pillar.get']('gerrit:libraries', {}).items() %}
```

Comment:
As we marge pillars in map.jinja we should take it from **settings.libraries** not directly from pillars